scripts/_ecs.sh

```diff
@@ -168,7 +168,7 @@ function _ecs_register_new_image_for_service() {
     --service ${service_name} \
     --task-definition ${new_task_definition_arn} > /dev/null
 
-  _ecs_deregister_stale_task_definitions
+  _ecs_deregister_stale_task_definitions ${service_name}
```

Comment:
Can't deregister task definitions if we don't know which service to look at. Doh